build/scripts/cleanup_perf.ps1

```diff
@@ -2,62 +2,90 @@ set-variable -name LastExitCode 0
 set-strictmode -version 2.0
 $ErrorActionPreference="Stop"
 
+param (
+    [string]$CPCLocation = "C:/CPC"
+)
+
 # If the test runner crashes and doesn't shut down CPC, CPC could fill
 # the entire disk with ETL traces.
-try {
-    taskkill /F /IM CPC.exe 2>&1 | Out-Null
+
+function KillAndIgnore {
+    try {
+        taskkill /F /IM $args[0] 2>&1 | Out-Null
+    }
+    catch {}
 }
-catch {}
+
+KillAndIgnore Cpc.exe
+KillAndIgnore msbuild.exe
+KillAndIgnore csc.exe
+KillAndIgnore vbc.exe
+KillAndIgnore VBCSCompiler.exe
+
 try {
-    taskkill /F /IM msbuild.exe 2>&1 | Out-Null
+    & $CPCLocation/cpc.exe /Stop /SkipClean 2>&1 | Out-Null
 }
 catch {}
-try {
-    taskkill /F /IM VBCSCompiler.exe 2>&1 | Out-Null
+
+if (Test-Path ToArchive) {
+    Remove-Item -Recurse -Force ToArchive
 }
-catch {}
 
-# Move all etl files to the a folder for archiving
-echo "creating ToArchive directory"
-mkdir ToArchive
-echo "moving C:/CPC/DataBackup* to ToArchive"
-mv C:/CPC/DataBackup* ToArchive
-ls ToArchive
+if (Test-Path $CPCLocation) {
+    # Move all etl files to the a folder for archiving
+    echo "creating ToArchive directory"
+    mkdir ToArchive
+    echo "moving C:/CPC/DataBackup* to ToArchive"
+    mv C:/CPC/DataBackup* ToArchive
+    ls ToArchive
+}
```

Comment:
Not as it is right now.  Currently cleanup_perf is run *before* and after perf_run.  This means that if the post-run cleanup works, there will not be a CPC location.

I should probably have "before or after" be a parameter and use that to decide if I should archive or not.